tv_apps/smart-system/js/update_manager.js

```diff
@@ -508,8 +511,12 @@ var UpdateManager = {
     var _ = navigator.mozL10n.get;
     var units = ['bytes', 'kB', 'MB', 'GB', 'TB', 'PB'];
 
-    if (!bytes)
+    if (!bytes) {
+      var size = {
+        var 
```

Comment:
According to the code in [1], we don't need to modify here.

[1] https://github.com/mozilla-b2g/gaia/blob/1b71c4cb311aa3cc9d61bc44a495cbc96af938b6/apps/system/js/update_manager.js#L853-L857